Cargo.toml

```diff
@@ -19,6 +19,8 @@ i18n-embed-fl = "0.6"
 rust-embed = "6"
 paste = "1.0"
 palette = "0.7"
+libsecret = { version = "0.4.0", features = ["v0_20"] }
+gio = "0.18.4"
```

Comment:
We shouldn't depend on GNOME libraries.

Reply:
I looked around a bit for alternatives, but it seems that there are not many good options available. Do you have any alternatives you prefer?

Reply:
Does cosmic desktop have any current or planned keyring?

Reply:
A crate will need to be developed that provides zbus bindings for https://specifications.freedesktop.org/secret-service/latest/.

Reply:
Well, that expands the scope of this password manager quite a lot 😅

Reply:
I see that zbus is using async_io::block_on for its blocking wrappers. So, would it be an ok solution to use the async secret-service library with a similar blocking wrappers?

Reply:
See https://github.com/pop-os/dbus-settings-bindings for existing zbus bindings we have.

Reply:
There is no need. zbus lets you choose a runtime, and libcosmic has a tokio feature.

Reply:
https://crates.io/crates/secret-service
This has a `rt-tokio-crypto-rust` feature.

Reply:
I just saw that they provide a blocking API also, so I will probably not have to explicitly depend on tokio....

Reply:
libcosmic already uses tokio for its runtime, so it would make sense to use it.

Reply:
I have now replaced libsecret with the secret-service crate.